specification/cognitiveservices/data-plane/Face/preview/v1.0-preview/Face.json

```diff
@@ -5218,7 +5236,8 @@
       "properties": {
         "personId": {
           "format": "uuid",
-          "type": "string"
+          "type": "string",
+          "x-nullable": false
```

Comment:
In the future, `x-nullable:false` is the default and can be omitted.

Reply:
I had to explicitly add it in order for AutoRest to generate a property without nullable--we needed type Guid instead of Guid? there.

Reply:
I see.  No need to block on this then but that seems weird.

@mikekistler do you know if this is a bug in the C# codegen?